test_linux.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+timeout 20s ./zubax-kucher
+exit_status=$?
+if [ $exit_status -eq 124 ]; then
+    echo No error found
```

Comment:
```suggestion
```

This doesn't really matter since this condition needs to be removed anyway, I just want to say that, typically, a well-behaved command-line tool reports nothing on success.